app/(tabs)/(home)/components/carousel/EachCarousel.tsx

```diff
@@ -0,0 +1,89 @@
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { ImageBackground, SafeAreaView, Text, useWindowDimensions, View } from 'react-native';
+
+import colors from '@/app/global/colors';
+
+import EachCategory from './EachCategory';
+
+const EachCarousel = ({
+  title,
+  description,
+  image,
+  // TODO: This will be in use not now
```

Comment:
Use better TODO texts: 

```suggestion
  // TODO: This will be used later
```